tests/integration/RunIntegrationTest.cmake

```diff
@@ -12,5 +12,5 @@ execute_process(COMMAND $ENV{ES_INTEGRATION_PREFIX} "${ES}" --config "${TEST_CON
     RESULT_VARIABLE TEST_RESULT)
 
 if(TEST_RESULT)
-    message(FATAL_ERROR "Integration test failed with:\n${TEST_OUTPUT}")
+    message(FATAL_ERROR "Integration test failed with '${TEST_RESULT}':\n${TEST_OUTPUT}")
```

Comment:
Only thing: I feel like the message could be clearer what is the result and what is the output if you know what I mean.

Reply:
I mean within the message, but maybe that's unnecessary.
Something along
result:
output:
now that I look at it it seems pretty obvious though 😅